tests/test_view.py

```diff
@@ -108,3 +110,18 @@ def test_slicing() -> None:
 
     for arr_np, arr_rle in zip(arrays_np, arrays_rle):
         npt.assert_array_equal(arr_np, arr_rle)
+
+
+def test_anchor_ref() -> None:
+    try:
+        gc.disable()
+        gc.collect()
+
+        n_objects_pre = len([o for o in gc.get_objects() if isinstance(o, RLEArray)])
+
+        RLEArray._from_sequence(np.arange(10))
+
```

Comment:
Should you also check if there is an anchor object in the GC? wouldn't that be more explicit?

Reply:
I don't care about the anchor that much since it doesn't contain much memory, but you're right, probably both should be checked.